source/indexes/create-vector-search-index.txt

```diff
@@ -0,0 +1,142 @@
+
+- Hosted on `MongoDB Atlas <https://www.mongodb.com/docs/atlas/>`__ and
+  have an Atlas cluster tier of M10 or higher.
+
+- A local deployment that is set up using the `Atlas CLI
+  <https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/>`__.
+
+Additionally, your deployment must run MongoDB version 7.0 or later.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+   
+   .. step:: Open the index creation dialog
+
+      From the :ref:`Indexes <indexes-tab>` tab, click the
+      :guilabel:`Create` button, then click :guilabel:`Search Index`.
+
+   .. step:: Specify a name for the index
+
+   .. step:: Select the Atlas Search Index type
+
+      Select :guilabel:`Vector Search`.
+
+   .. step:: Provide the Atlas Vector Search Index configurations
+
+      You must initially provide the following default vector search index 
+      configurations. You can modify this later.
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 20 10 70
+
+         * - Field
+           - Type
+           - Description
+
+         * - ``path``
+           - string
+           - The field name to index.
+        
+         * - ``numDimensions``
+           - int 
+           - The number of vector dimensions, which Atlas Search enforces at index- and 
+             query-time. This value can't be greater than 2048.
+        
+         * - ``similarity``
+           - string
+           - The vector similarity function used to search for top K-nearest neighbors.
+             Select from the following:
+
+             .. list-table::
+                :header-rows: 1
+                :widths: 50 50
+
+                * - Function
+                  - Description
+
+                * - ``euclidean``
+                  - A function that measures the distance between ends of vectors. This allows 
+                    you to measure similarity based on varying dimensions. 
+
+                * - ``cosine``
+                  - A function that measures similarity based on the angle between vectors. This 
+                    allows you to measure similarity that isn't scaled by magnitude. You can't use 
+                    zero magnitude vectors with cosine. To measure cosine similarity, we recommend 
+                    that you normalize your vectors and use dotProduct instead.
+
+                * - ``dotProduct``
+                  -  A function that measures similar to cosine, but takes into account the 
+                     magnitude of the vector. This allows you to efficiently measure similarity 
+                     based on both angle and magnitude. To use dotProduct, you must normalize 
+                     the vector to unit length at index- and query-time.
+
+   .. step:: Click :guilabel:`Create Search Index`
+
+Example
+-------
+
+The following example definition uses the ``sample_mflix.embedded_movies`` collection 
+and indexes the ``plot_embedding`` field: 
```

Comment:
[suggestion] I would suggest adding more context around what the example shows

```suggestion
The following example definition uses the ``sample_mflix.embedded_movies`` collection 
and indexes the ``plot_embedding`` field. This code block illustrates an example Atlas Vector Search configuration
```

Reply:
Updated to something similar.